### DISCUSSION TITLE
Full Screen Image Widget / Screen Specific Background

### DISCUSSION THREAD

**Discussion by Durahl** [EdgeTX/edgetx#3658] [Ideas]
Greetings!

I just followed the Dev Chat and was asked to post my Request for an Idea on the GitHub

I keep using my EdgeTX Radio for my [Collection of RC Tanks](https://i.imgur.com/f1JczdA.jpg) using different Controller Brands, Controller Generations, and Controller Capabilities. Some models can perform like a handful of easy to remember Functions while others will essentially slam you with a Driver's License load of Possibilities to memorize 😆

Pairing this much Variation with my infrequent use of these Models I'll usually end up not recalling to which Switch a Function was bound to hence my Idea of a Full Screen Image Widget ( or a Screen specific Background Image I could add to Screen 2 or 3 ) one can take a glance on to remember where things are.

In my case I'm using a FrSky Horus X12S and I'd be populating said Widget ( or the Background Image ) with the CAD Image of the X12S and scribble some notes onto it:
![IMG](https://i.imgur.com/3nEFsLA.png)

Right now, I'm using the [Switch2 Lua Script](https://www.rcgroups.com/forums/showpost.php?p=48657103&postcount=3966) to help me identify which Switches are Populated to begin with but I'm fearing for its long term usability should it fall victim to an Version Upgrade incompatibility down the Roadmap.

Thanks for your consideration! 😁

**Comment by pfeerick** [comment]
Is this something the Model Notes capability would be useful for? There seems to be a gap in the user guide about it, probably because it only gets shown in the main menu when model notes are present, but I think John's video about from OpenTX should still apply today. Just for us, to manually show them, it appears as an extra icon in the main menu carousel.  https://www.youtube.com/watch?v=CoaAzHhf6w4

**Comment by Durahl** [comment]
Model Notes would certainly be a last resort for me should the Switch2 Script lose functionality down the road, but IMHO not be the preferred solution.
Ideally, I'd like to see a native implementation of what the Switch2 Script does ( though slightly different ) but that is asking for a little bit much so being able to assign a Screen Filling secondary Background Image the user can do with whatever they want to on one of the Screens would probably be the next best low-cost solution.

**Reply by pfeerick** [reply]
You might be interested in this widget then : https://github.com/EdgeTX/lua-scripts#vu-fullscreen-image-viewer-widget-for-big-screens

That will at least do the full-screen image part of things if you set the screen up correctly.

**Comment by jrwieland** [comment]
With a quick serch, it seems that someone else has a similar issue and wrote a widget that can display what your after.
Below is a link to their site and the zip file is attached.  
to use it create an image that matches you models name and the script will look for the matching name and show the image that you create using CAD with the notes on the image.
 
https://colinsradiocontrol.com/index.php/lua-scripts/14-t16widgets

[InfoCard.zip](https://github.com/EdgeTX/edgetx/files/11691732/InfoCard.zip)

**Comment by Durahl** [comment]
Thanks to both of you!
In the end my Vote went with the solution suggested by @pfeerick as it seemed to allow for a bit more control over the Widget.
The "Usage" Portion was a bit confusing at first but in the end, I got over that problem too 😅

**Comment by marcoose** [comment]
Late to the party, but I had exactly the same interest as you.  I found out about the existing widget too late and already wrote my own, but if you are looking for a tool to make the "documentation screens" I put together a basic web app that lets you label all your switches:  https://marcoose.github.io/tx16s-docs/

e.g.
![Whoopfly](https://github.com/user-attachments/assets/56f6c1ba-b4ef-4ca1-b21e-ed5cb9ffa951)
